flake.nix

```diff
@@ -119,8 +119,8 @@
           gaia6 = mkApp { name = "gaia"; drv = packages.gaia6_0_4; exePath = "/bin/gaiad"; };
           gaia6_0_2 = mkApp { name = "gaia"; drv = packages.gaia6_0_2; exePath = "/bin/gaiad"; };
           gaia6_0_3 = mkApp { name = "gaia"; drv = packages.gaia6_0_3; exePath = "/bin/gaiad"; };
```

Comment:
Note that Gaia v4 is deprecated because SDK v0.42 reached end of life.

Reply:
Oh, good to know. Removed.